exporter/postgres/setup_pg10.sql

```diff
@@ -1,5 +1,12 @@
 DO $$
 BEGIN
+    -- The pgmonitor role is required by the pgnodemx extension in PostgreSQL versions 9.5 and 9.6
+    -- and should be removed when upgrading to PostgreSQL 10 and above.
+    IF EXISTS (SELECT 1 FROM pg_roles WHERE rolname = 'pgmonitor') THEN
+        REVOKE pgmonitor FROM ccp_monitoring;
```

Comment:
It's possible that `ccp_monitoring` doesn't exist yet, though unlikely. It might be safer to move this block after the one on L10-12.

Reply:
missing grant

Reply:
This is a good point

Reply:
That makes sense, I'll make that update.